retail/clickstream/README.md

```diff
@@ -0,0 +1,148 @@
+# Click stream data - Data Layer to Pub/Sub
```

Comment:
I think we need something more inline with the app for the file location.
We currently have retail/retail-java-application can we make this something like retail/retail-<something>-application.

Reply:
Changed to retail-clickstream-application